app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -69,8 +79,20 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                     ])
                     ->get()
                     ->toArray(),
+                'articleVote' => ArticleWasHelpful::active() ? optional(
+                    $article->knowledgeBaseArticleVotes()
+                        ->where('user_id', $userid)
+                        ->where('user_type', $userType)
+                        ->select([
+                            'id',
+                            'is_helpful',
+                        ])
+                        ->first()
+                )->toArray() : [],
             ]),
             'portal_view_count' => $article->portal_view_count,
+            'guest_id' => Session::get('guest_id'),
```

Comment:
What is the purpose of passing the `guest_id` here? If you need to retrieve it somewhere later in the code you should be able to reference it in the session right?

Reply:
Done